src/Sulu/Bundle/SnippetBundle/Admin/SnippetAdmin.php

```diff
@@ -226,26 +226,28 @@ public function configureViews(ViewCollection $viewCollection): void
         if ($this->activityViewBuilderFactory->hasActivityListPermission() || $this->referenceViewBuilderFactory->hasReferenceListPermission()) {
             $insightsResourceTabViewName = SnippetAdmin::EDIT_FORM_VIEW . '.insights';
 
-            $viewCollection->add(
-                $this->viewBuilderFactory
-                    ->createResourceTabViewBuilder($insightsResourceTabViewName, '/insights')
-                    ->setResourceKey(SnippetDocument::RESOURCE_KEY)
-                    ->setTabOrder(6144)
-                    ->setTabTitle('sulu_admin.insights')
-                    ->setTitleProperty('')
-                    ->setParent(SnippetAdmin::EDIT_FORM_VIEW)
-            );
-
-            if ($this->activityViewBuilderFactory->hasActivityListPermission()) {
+            if ($this->securityChecker->hasPermission(static::SECURITY_CONTEXT, PermissionTypes::EDIT)) {
```

Comment:
this can directly be added to line `226` if statement as `if ( (current) && (this))` so no additional nesting if is required to show this detail tabs.